lib/graphql/subscriptions/event.rb

```diff
@@ -65,12 +65,24 @@ def stringify_args(arg_owner, args)
 
               if arg_defn
                 normalized_arg_name = camelized_arg_name
-                next_args[normalized_arg_name] = stringify_args(arg_defn.type, v)
               else
                 normalized_arg_name = arg_name
                 arg_defn = get_arg_definition(arg_owner, normalized_arg_name)
               end
-              next_args[normalized_arg_name] = stringify_args(arg_defn.type, v)
+              arg_base_type = arg_defn.type
+              while arg_base_type.respond_to?(:of_type)
```

Comment:
I struggled here, as this does not feel right, and that there must be some way to "unwrap" the low-level type that would be desired here; I was not able to easily find it, though, so if there is a better way to get at what I'm after happy to change it. 

I also don't know if I have a guarantee that the type chain here in fact will ever terminate and am slightly terrified this could cause an infinite loop, so happy to switch or add safeguards as well.